books/kestrel/apt/isodata-doc.lisp

```diff
@@ -382,6 +383,23 @@
        while the sections with `Predicate' in their title
        refer to the case in which @(':predicate') is @('t')."))
 
+    (xdoc::desc
+     "@(':undefined') &mdash; default @(':auto')"
+     (xdoc::p
+      "Denotes the value that the generated new function must return
+       outside of the new domain.")
+     (xdoc::p
+      "It must be one of the following:")
+     (xdoc::ul
+      (xdoc::li
+       "@(':auto'), to use @('nil') or @('(mv nil ... nil)') for single-value
+        and multi-value functions respectively.")
+      (xdoc::li
+       "Any other term. It must be a term that only references logic-mode
+        functions and that includes no free variables other than
+        @('x1'), ..., @('xn'). This term must have no output
+        @(see acl2::stobj)s. This term must not reference @('old').")))
+
```

Comment:
Just after the two `li` bullets, as last part of this `xdoc::desc`, we could add a paragraph `(xdoc::p ...)` saying that one can use a quoted `':auto` if they want to return the keyword `:auto` as undefined result.